share/smack/include/math/math.h

```diff
@@ -8,11 +8,11 @@
 float fabsf(float x);
 float fdimf(float x, float y);
 float roundf(float x);
+long lroundf(float x);
 //The following 3 functions are incomplete pending rounding mode implementation
```

Comment:
Now that we have rounding mode working, shouldn't this comment be addressed as well?